sdcm/nosql_thread.py

```diff
@@ -55,7 +58,27 @@ def _run_stress(self, loader, loader_idx, cpu_idx):
         LOGGER.debug("'running: %s", stress_cmd)
         with NoSQLBenchStressEvent(node=loader, stress_cmd=stress_cmd, log_file_name=log_file_name) as stress_event:
             try:
-                return loader.remoter.run(cmd=f'docker run {self._nosqlbench_image} {stress_cmd}',
+                # copy graphite-exporter config file to loader
+                loader.remoter.send_files(src=self.GRAPHITE_EXPORTER_CONFIG_SRC_PATH,
+                                          dst=self.GRAPHITE_EXPORTER_CONFIG_DST_PATH,
+                                          verbose=False)
+
+                # create shared network for the containers
+                create_network_cmd = "docker network create --driver bridge nosql"
+
+                graphite_run_cmd = "docker run -d -p 9108:9108 -p 9109:9109 -p 9109:9109/udp " \
+                                   "-v /tmp/graphite_mapping.conf:/tmp/graphite_mapping.conf " \
+                                   "--name=graphite-exporter --network=nosql " \
+                                   "prom/graphite-exporter --graphite.mapping-config=/tmp/graphite_mapping.conf"
+                loader.remoter.run(cmd=create_network_cmd)
```

Comment:
It will fail on seconds run, since graphite is already running.
Also you will have problems with running it in parallel.
I believe we should make graphite part of the nosqlbench docker image , let me know if you need help with this

Reply:
_It will fail on seconds run, since graphite is already running._ I will add an `ignore_status` to this run command.

_Also you will have problems with running it in parallel._ If you mean running multiple instances on one loader, than the question would be - why are we doing that? If we want more threads / async, we'd rather tweak the CLI options given to NB, I think. But maybe I'm missing some use case here.

_I believe we should make graphite part of the nosqlbench docker image , let me know if you need help with this_ We can discuss that. I'm a bit weary of putting to many things in one image. We could also use `docker-compose` to have them work together, i didn't see that as necessary just for 2 containers we're using right now.

Reply:
I've added a `ignore_status` argument to the `run` remoter method call to avoid erroring out on graphite-exporter already running.
As for making graphite part of the nosqlbench image,  I think it might introduce more confusion than  advantages (e.g. we'll have a very long list of options for both graphite-exporter and nosqlbench, stringed together in one command). That said I'm open to trying it out, though maybe not in this PR.